### PR TITLE
Added ANT build file for local GAE. Added default index.html.

### PR DESCRIPTION
**Pull request by jesuscmartinez**
Added support for local GAE via Ant, based off of the template in appengin-java-sdk-1.9.17.

Added default index.html. I couldn't find the default in the pages directory. Just copy and pasted the default from the public running instance minus the version info. I don't know the best way to pull this data from the project. Should it be a property file, version.java file or introduce it into the build progress?

If these changes make it in, then I can dockerize based off of the google master branch, rather then my branch.

Thanks

### CODE REVIEW

**Comment by paradoxengine** [comment]
Thank you! This is a useful change and I'm going to approve it. However I need to go through some refactoring steps internally before I can accept pulls - I should be done by the end of the week. Please also check the CONTRIBUTING note if you have not done so already.

**Comment by laixb** [comment]
hi 
how can i build this project?

**Comment by paradoxengine** [comment]
I'll be happy to accept the pull Jesus - I had to do a bit of refactoring to enable that, but now we're ready.
Can you please check if the pull can be merged and have a look at the CONTRIBUTING notes? There is a bit of formality to go through but it should take 5 minutes.

**Comment by jesuscmartinez** [comment]
I found that you added an index.html page to the WEB-INF folder however I don't believe those file are made publicly accessible. At least not when running locally with appengine-java-sdk-1.9.20. So I removed the copy of the index.html from the pages folder and add a step in the build.xml to copy the index.html from the WEB-INF folder.

It should be good to go. Thanks

File: build.xml [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+  </target>
+
+  <target name="enhance" depends="compile"
+          description="Enhance the classes after compilation">
+    <enhance_war war="${war.dir}"/>
+  </target>
+
+  <target name="getjars">
+ 		<mkdir dir="${war.dir}/WEB-INF/lib" />
+		<get src="http://central.maven.org/maven2/com/google/guava/guava/18.0/guava-18.0.jar" dest="${war.dir}/WEB-INF/lib"/>
```
Comment (paradoxengine) [inline build.xml]: Can we use https links please?

File: build.xml [hunk 2]
```diff
@@ -0,0 +1,148 @@
+<!-- This is a Ant file for Google AppEngine project -->
```
Comment (paradoxengine) [inline build.xml]: Maybe let's use a specific comment re firing range?

**Comment by paradoxengine** [comment]
This looks perfect, thank you.
I have 2 minor comments, other than that I can merge - pending one thing though. As per the CONTRIBUTING file I have to ask you to go through the cla (https://cla.developers.google.com/about/google-individual). Should be a no-op for you essentially, but still :-( sorry.

**Comment by jesuscmartinez** [comment]
I've made the suggested changes. I also accepted the agreement back in January. 
screen shot 2015-05-22 at 3 52 17 pm

**Comment by paradoxengine** [comment]
Fantastic, thank you.

**Pull request merged**